conbench/api/_google.py

```diff
@@ -1,44 +1,72 @@
 import json
+import logging
 import os
+import time
 
 import flask as f
 import requests
 
+from ..config import Config
 
-def get_google_config():
+log = logging.getLogger(__name__)
+
+# Rely on INTENDED_BASE_URL to have a trailing slash.
+OIDC_CALLBACK_URL = Config.INTENDED_BASE_URL + "api/google/callback"
```

Comment:
Here we make use of [the ](https://github.com/conbench/conbench/blob/d4030f80bbfba0484104d01db3124c75206ba2aa/conbench/config.py#L50)